common/federation/src/main/java/app/coronawarn/server/common/federation/client/FederationFeignHttpClientProvider.java

```diff
@@ -67,24 +64,23 @@ public FederationFeignHttpClientProvider(FederationGatewayConfig config) {
   @Bean
   public Client createFeignClient() {
     return new ApacheHttpClient(
-        federationHttpClientFactory(connectionPoolSize, keyStore, keyStorePassword, "JKS")
+        federationHttpClientFactory(connectionPoolSize, keyStore, keyStorePassword)
             .createBuilder().build());
   }
 
   /**
    * Creates an {@link ApacheHttpClientFactory} that validates SSL certificates but no host names.
    */
   private ApacheHttpClientFactory federationHttpClientFactory(int connectionPoolSize, File keyStorePath,
-                                                              String keyStorePass,
-                                                              String certificateType) {
+      String keyStorePass) {
     return new DefaultApacheHttpClientFactory(HttpClientBuilder.create()
         .setMaxConnPerRoute(connectionPoolSize)
         .setMaxConnTotal(connectionPoolSize)
-        .setSSLContext(getSslContext(keyStorePath, keyStorePass, certificateType))
```

Comment:
So,  does the certificate type not have to be passed anymore?

Reply:
Yes, as the parameter in `getSslContext` was unused anway